apps/app/src/app/pages/(examples)/examples/authentication/(authentication).page.ts

```diff
@@ -0,0 +1,64 @@
+import { RouteMeta } from '@analogjs/router';
+import { Component } from '@angular/core';
+import { HlmButtonDirective } from '@spartan-ng/ui-button-helm';
+import { HlmCardDirective } from '@spartan-ng/ui-card-helm';
+import { metaWith } from '~/app/shared/meta/meta.util';
```

Comment:
@danilolmc - this tilde seems to be causing the build to fail, I think this just needs to be updated to "@spartan-ng/app/shared/meta/meta.util" instead.